include/sdf/Box.hh

```diff
@@ -65,6 +68,12 @@ namespace sdf
     /// \return A reference to a gz::math::Boxd object.
     public: gz::math::Boxd &Shape();
 
+    /// \brief Calculate and return the Mass Matrix values for the Box
+    /// \param[in] density Density of the box in kg/m^3
```

Comment:
```suggestion
    /// \param[in] _density Density of the box in kg/m^3
```

Reply:
Corrected in [1f16e1f](https://github.com/gazebosim/sdformat/pull/1304/commits/1f16e1f0a5454acc5a30a4b86b8f273b1535eb95)